packages/block-editor/src/components/image-link-destinations/style.native.scss

```diff
@@ -0,0 +1,16 @@
+// used in both light and dark modes
+.placeholderTextColor {
+	color: #87a6bc;
+}
+
+.optionIcon {
+	color: $blue-50;
+}
+
+.optionIconDark {
+	color: $blue-30;
+}
+
+.unselectedOptionIcon {
+	opacity: 0;
+}
```

Comment:
It appears these selectors in the original stylesheet are now unused and can be removed. Note that `.linkSettingsPanel` remains in use. 

https://github.com/WordPress/gutenberg/blob/8a4ddf2d506b217895768adf44e024446ce08532/packages/components/src/mobile/link-settings/style.native.scss#L6-L21